tests/typescript/conformance/classes/classDeclarations/classAbstractKeyword/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,1023 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`classAbstractAccessor.ts 1`] = `
+
+abstract class A {
+   abstract get a();
+   abstract get aa() { return 1; }
+   abstract set b(x: string);
+   abstract set bb(x: string) {}
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {
+  abstract get a();
+  abstract get aa();
+  abstract set b(x: string);
+  abstract set bb(x: string);
+}
+
+`;
+
+exports[`classAbstractAsIdentifier.ts 1`] = `
+class abstract {
+    foo() { return 1; }
+}
+
+new abstract;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+class abstract {
+  foo() {
+    return 1;
+  }
+}
+
+new abstract();
+
+`;
+
+exports[`classAbstractAssignabilityConstructorFunction.ts 1`] = `
+abstract class A { }
+
+var AAA: new() => A;
+
+AAA = A;
+AAA = "asdf";~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {}
+
+var AAA: new() => A;
+
+AAA = A;
+AAA = "asdf";
+
+`;
+
+exports[`classAbstractClinterfaceAssignability.ts 1`] = `
+
+var I: IConstructor;
+
+abstract class A {
+    x: number;
+    static y: number;
+}
+
+var AA: typeof A;
+AA = I;
+
+var AAA: typeof I;
+AAA = A;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+var I: IConstructor;
+
+abstract class A {
+  x: number;
+  static y: number;
+}
+
+var AA: typeof A;
+AA = I;
+
+var AAA: typeof I;
+AAA = A;
+
+`;
+
+exports[`classAbstractConstructor.ts 1`] = `
+abstract class A {
+    abstract constructor() {}
+}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {
+  constructor() {}
```

Comment:
This one is also an AST bug. The parser detects this constructor as type `"MethodDefinition"`, kind `"constructor"`.